README.md

```diff
@@ -591,6 +594,8 @@ use Zenstruck\Foundry\Proxy;
 
 /**
  * @method static Post|Proxy findOrCreate(array $attributes)
+ * @method static Post|Proxy random()
+ * @method static Post|Proxy randomSet(int $min, ?int $max = null)
```

Comment:
documenting this is tricky because `randomSet(5)` means "give me 5", so it's not a `$min` in that situation. I'm not sure exactly what to call it though - that method really wears two hats. I'm also not sure if we split it into 2 separate methods, what each would be called.

Also, this would return `Post[]|Proxy[]`

Reply:
> I'm also not sure if we split it into 2 separate methods, what each would be called.

Yeah... `randomSet()` and `randomRange()`?

> Also, this would return Post[]|Proxy[]

Ahh, good catch!